cli/internal/configurator/configurator.go

```diff
@@ -77,9 +79,21 @@ func resolveConfigFile(v *viper.Viper) (string, error) {
 	if v == nil {
 		return "", errors.New("[Error] Config is malformed. Please delete your .configuration file in your $HOME/tython directory and try again")
```

Comment:
```suggestion
		return "", errors.New("[Error] Config is malformed. Please delete your .configuration file in your $HOME/tython directory and try again")
```

Will this print the string literal of "$HOME" or the full path?

Reply:
nope just "$HOME"

Reply:
Go can interpolate strings right?

Reply:
pushed an update that removes the reference to $HOME, my guess is that in most error conditions where we can't get the working directory we will also have trouble resolving the home directory so left it vague.  We are remove the statement as well.

Reply:
Left the original error the same as we had it, I would have to look into what conditions would cause us to get there

Reply:
It was simply more-so to tell the user we are storing it in their $HOME directory, whatever that is. It can be changed to actually resolve if necessary.